common/management/commands/init_db.py

```diff
@@ -0,0 +1,239 @@
+        ).date()
+        end = datetime.datetime.strptime(
+            f"{date.year}-07-31" if summer else f"{date.year + 1}-01-31", "%Y-%m-%d"
+        ).date()
+        semester, _ = Semester.objects.get_or_create(
+            begin=begin,
+            end=end,
+            year=f"{date.year}",
+            winter=not summer,
+            active=i == 0,
+            inbus_semester_id=124 + i,
+        )
+        semesters.append(semester)
+        if summer:
+            date = date.replace(month=8)
+            date = date.replace(year=date.year - 1)
+        else:
+            date = date.replace(month=3)
+    return semesters
+
+
+def create_subject(name: str, abbr: str) -> Subject:
+    subj, _ = Subject.objects.get_or_create(name=name, abbr=abbr)
+    return subj
+
+
+def create_class(
+    code: str, teacher: User, semester: Semester, subject: Subject, day: Class.Day, time: datetime
+) -> Class:
+    created_class, _ = Class.objects.get_or_create(
+        code=code, teacher=teacher, semester=semester, subject=subject, day=day, time=time
+    )
+    return created_class
+
+
+def create_random_class(teacher: User, semesters: list[Semester], subjects: list[Subject]) -> Class:
+    code = f"{random.choice(string.ascii_uppercase)}/{''.join(random.choices(string.digits, k=2))}"
+    chosen_semester = random.choice(semesters)
+    chosen_subject = random.choice(subjects)
+    day = random.choice(Class.Day.choices)[0]
+    time = datetime.time(hour=random.choice(range(7, 19)), minute=random.choice(range(0, 60, 15)))
+    return create_class(code, teacher, chosen_semester, chosen_subject, day, time)
+
+
+def create_task(subject: Subject, semester: Semester, name_suffix: int) -> Task:
+    name = f"Task_{subject.abbr}_{name_suffix}"
+    code = "/".join([subject.abbr, str(semester), name.lower()])
+    created_task, _ = Task.objects.get_or_create(
+        name=name,
+        code=code,
+        subject=subject,
+        announce=True,
+        plagiarism_key=None,
+        type=random.choice(Task.TaskType.choices)[0],
+    )
+
+    return created_task
+
+
+def assign_task(task: Task, clazz: Class, semester: Semester) -> AssignedTask:
+    assigned_date = semester.begin + datetime.timedelta(
+        days=random.randint(0, (semester.end - semester.begin).days)
+    )
+    assigned_date = datetime.datetime.combine(assigned_date, datetime.time.min)
+    assigned_task, _ = AssignedTask.objects.get_or_create(
+        task_id=task.pk,
+        clazz_id=clazz.pk,
+        defaults={
+            "assigned": timezone.make_aware(assigned_date),
+            "deadline": timezone.make_aware(assigned_date + datetime.timedelta(days=7)),
+            "max_points": random.randint(1, 5),
+        },
+    )
+
+    return assigned_task
+
+
+@dataclass
+class FileToCreate:
+    name: str
+    contents: str
+
+
+def create_task_files() -> list[FileToCreate]:
+    files = []
+    file_names = [".taskid", "config.yaml", "readme.md"]
```

Comment:
```suggestion
    file_names = [".taskid", "config.yml", "readme.md"]
```
We hardcode this filename in a bunch of places.